internal/acceptance/tools/tools.go

```diff
@@ -1,10 +1,10 @@
 package tools
 
 import (
-	"crypto/rand"
```

Comment:
cryptographically secure randomness is not required here. and the `math/rand` package has lots more convenient functions